app/home/home.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {provide} from 'angular2/core';
+import {it, describe, expect, injectAsync, beforeEachProviders} from 'angular2/testing';
+import {TestComponentBuilder, ComponentFixture, setBaseTestProviders} from 'angular2/testing';
```

Comment:
Hmm... yeah, that's not great.  And I don't see any "all" importers.  Part of my frustration with this component test is the sheer number of things that have to be imported.

You'd prefer a wrapped line then?  Works for me.